licenses/cachetools.txt

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
I think it is not necessary to have this license folder, listed in `LICENSE` and `LICENSE.bin` should be enough.